.github/workflows/generate_calamari_weights_files.yml

```diff
@@ -224,6 +224,22 @@ jobs:
               id: pallet_preimage
               name: pallet_preimage
             iterations: 20
+          -
+            extrinsic:
+              id: '*'
+              name: pallet_assets
+            pallet:
+              id: pallet_assets
+              name: pallet_assets
+            iterations: 20
+          -
+            extrinsic:
+              id: '*'
+              name: pallet_asset_manager
+            pallet:
+              id: pallet_asset_manager
+              name: pallet_asset_manager
+            iterations: 20
```

Comment:
these changes weren't there before your force push - are they intended? If so, why weren't they merged to manta branch instead

Reply:
I remember I didn't change anything about this file.

Reply:
This is from manta branch